src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+$field
+$field-hover
+$border-subtle
+$border-subtle-selected
+$border-strong
+```
+--->
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="White">
+
+![Layer set tokens applied to dropdown in the White theme](images/color-implementation-contextual-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![Layer set tokens applied to dropdown in the Gray 10 theme](images/color-implementation-contextual-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![Layer set tokens applied to dropdown in the Gray 90 theme](images/color-implementation-contextual-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![Layer set tokens applied to dropdown in the Gray 100 theme](images/color-implementation-contextual-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+<br />
+
+#### Example of how to apply contextual layers tokens in a layout
+
+Referencing the image below, the starting base level is still the page area
+behind and above the tabs; it uses `$background`. However, there are no sets so
+with each level added simply reuse the same tokens. The tabs and the tile live
+on different levels but they both use the `$layer` token that will output a
+different values in code with the help of the layer component.
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="White">
+
+![In context contextual token example in the White theme](images/color-implementation-contextual-2-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![In context contextual token example in the Gray 10 theme](images/color-implementation-contextual-2-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![In context contextual token example in the Gray 90 theme](images/color-implementation-contextual-2-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![In context contextual token example in the Gray 100 theme](images/color-implementation-contextual-2-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+#### Building components with contextual tokens
+
+Build a single component as you would normally but apply the contextual color
+tokens instead of the layer set tokens. Then in your product code use the layer
+component to express the nested layer visuals. Even if the component is never
+used at other layer levels it is still acceptable and encouraged to use the
+contextual tokens.
+
+Note: Contextual tokens are only available in code and are not a part of the
+design assets. Designer should use the layer set tokens when creating assets but
+can include contextual tokens in their specs and redlines.
+
+<Row>
+<Column colLg={8}>
+
+![Component spec example using contextual tokens](images/color-implementation-contextual-3.png)
+
+</Column>
+</Row>
+
+### Using the layer component
+
+The layer component is used with the contextual tokens to implement the layering
+model in code. The layer component and contextual tokens are essentially a short
+cut for development because the layer component will apply the correct values to
+a single component without having to develop explicit hard coded variants.
```

Comment:
Use the `Layer` component in order to have contextual tokens map to the correct layer value. Each token that is used to build a component rendered in a `Layer` component will automatically map to the next level based on the current layer set. These components can be nested up to three times, the last level corresponding to the 04 layer set of tokens. By default, tokens use the 01 layer set of tokens.